airbyte-integrations/connectors/source-okta/source_okta/custom_authenticators.py

```diff
@@ -2,9 +2,12 @@
 # Copyright (c) 2024 Airbyte, Inc., all rights reserved.
```

Comment:
Can you rename the file to `components.py`? It's not validated by any CI checks, but we'll consolidate custom components into that file for all connectors.

Reply:
Done